### PR TITLE
beta pages "Favorites", "My Posts" and "Messages" and corrected images

### PR DESCRIPTION
**Pull request by AnastasiiaMjlxy**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,10 +7,17 @@ import PostCreationPage from "./pages/PostCreationPage";
 import PostsPage from "./pages/PostsPage";
 import PostPage from "./pages/PostPage";
 import Page404 from "./pages/Page404";
+import FavoritesPage from "./pages/FavoritesPage";
+import MyPostsPage from "./pages/MyPostsPage";
+import MessagesPage from "./pages/MessagesPage";
```
Comment (ArCheeq) [inline src/App.js:12]: in the updated code now use lazy loading instead of default import

File: src/App.js [hunk 2]
```diff
@@ -20,9 +27,13 @@ function App() {
       <Routes>
         <Route path="/" element={<HomePage/>} />
         <Route path="/aboutUs" element={<AboutUsPage/>} />
+        <Route path="/messages" element={<MessagesPage/>} />
+        
         <Route element={<PrivateRoutes/>}>
           <Route path="/userProfile" element={<ProfilePage />} />
           <Route path="/createPost" element={<PostCreationPage/>} />
+          <Route path="/favorites" element={<FavoritesPage/>} />
+          <Route path="/myPosts" element={<MyPostsPage/>} />
```
Comment (ArCheeq) [inline src/App.js:36]: delete "/" symbol from path

File: src/pages/FavoritesPage.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import favorites from "../images/title/favorites.png";
+
+
+import Title from "../components/Title";
+import CardList from "../components/CardList";
+
+function FavoritesPage() {
+  return (
+    <main className="container max-w-xl mx-auto my-15 px-4">
```
Comment (ArCheeq) [inline src/pages/FavoritesPage.js:9]: remove tag "main", now it is in the layout component

File: src/pages/MyPostsPage.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import noticeboard from "../images/title/noticeboard.png";
+
+import plus from "../images/title/plus.png";
+import Title from "../components/Title";
+import MyCardList from "../components/MyCardList";
+import { Link } from "react-router-dom";
+
+function MyPostsPage() {
+  return (
+    <main className="container max-w-xl mx-auto my-15 px-4 flex flex-wrap justify-between gap-[15px] mt-8">
```
Comment (ArCheeq) [inline src/pages/MyPostsPage.js:10]: same

**Review by ArCheeq — CHANGES_REQUESTED** [review]
Everything is good, update the branch with changes and fix the comments please

File: src/components/messages/Done.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      <div className="w-full bg-gradient-to-br from-or-300 from-0% to-or-500
+                       to-100% rounded-3xl flex p-6">
+        <div className="flex flex-col justify-center align-center">
+            <img src={done} alt="done" style={{ width: '250px', height: '200px' }}/>
+            </div>
+        <div className="p-4 flex flex-col justify-center mx-auto">
+          <div className="flex items-center my-4">
+            <h2 className="text-4xl font-bold mb-2 text-left">Ми закрили збір!</h2>
+          </div>
+          <p className="text-gray-700 text-xl text-left my-4">Збір від Володимира Андрєєва був закритий! Дякуємо кожному, за допомогу. Тепер Ви можете перейти та подивитися на звіт автора.</p>
```
Comment (Misha999777) [inline src/components/messages/Done.js:16]: Line is too long

File: src/components/messages/MessagesList.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+function MessagesList() {
+  return (
+    <div className="flex flex-wrap justify-between gap-[15px] mt-8">
+      <Done/>
+      <Verified/>
+      <Issue/>
+      <Warning/>
+      <Views/>
+      
+
```
Comment (Misha999777) [inline src/components/messages/MessagesList.js:17]: Remove this gap

File: src/components/messages/MessagesList.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    </div>
+  );
+}
+
+export default MessagesList;
+
+
+
+
+
```
Comment (Misha999777) [inline src/components/messages/MessagesList.js:27]: Remove this gap

File: src/components/messages/MessagesList.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
+import Done from "./Done";
+import Verified from "./Verified";
+import Issue from "./Issue";
+import Warning from "./Warning";
+import Views from "./Views";
+
```
Comment (Misha999777) [inline src/components/messages/MessagesList.js:6]: One empty line is enough here

File: src/components/messages/Issue.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
+import error from "../../images/messagesImg/Error.png";
+
```
Comment (Misha999777) [inline src/components/messages/Issue.js:2]: One empty line is enough here

File: src/components/messages/Done.js [hunk 10]
```diff
@@ -0,0 +1,23 @@
+import done from "../../images/messagesImg/Done.png";
+
```
Comment (Misha999777) [inline src/components/messages/Done.js:2]: One empty line is enough here

File: src/components/MyCard.js [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+          <p className="font-normal text-100">
+            Кінцева сума:
+          </p>
+          <p className="text-100 text-2xl font-semibold">
+            1 000 000₴
+          </p>
+        </div>
+        
+        <div>
+  <div className="text-center mb-3 flex justify-between items-center">
```
Comment (Misha999777) [inline src/components/MyCard.js:37]: Align, please

File: src/components/MyCard.js [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+    <button className="font-normal text-100">Редагувати</button>
+    <button className="font-normal text-100">Видалити</button>
+  </div>
+</div>
+
+
```
Comment (Misha999777) [inline src/components/MyCard.js:43]: Remove this gap

File: src/components/MyCard.js [hunk 13]
```diff
@@ -0,0 +1,49 @@
+import car from "../images/car.png";
+
+
```
Comment (Misha999777) [inline src/components/MyCard.js:3]: One empty line is enough here

File: src/components/messages/Verified.js [hunk 14]
```diff
@@ -0,0 +1,28 @@
+import verified from "../../images/messagesImg/Verified.png";
+
```
Comment (Misha999777) [inline src/components/messages/Verified.js:2]: One empty line is enough here

File: src/components/messages/Views.js [hunk 15]
```diff
@@ -0,0 +1,22 @@
+import inf from "../../images/messagesImg/Attention.png";
+
```
Comment (Misha999777) [inline src/components/messages/Views.js:2]: One empty line is enough here

File: src/components/messages/Warning.js [hunk 16]
```diff
@@ -0,0 +1,23 @@
+import inf from "../../images/messagesImg/Attention.png";
+
```
Comment (Misha999777) [inline src/components/messages/Warning.js:2]: One empty line is enough here

File: src/pages/FavoritesPage.js [hunk 17]
```diff
@@ -0,0 +1,16 @@
+import favorites from "../images/title/favorites.png";
+
```
Comment (Misha999777) [inline src/pages/FavoritesPage.js:2]: One empty line is enough here

File: src/pages/MessagesPage.js [hunk 18]
```diff
@@ -0,0 +1,12 @@
+import MessagesList from "../components/messages/MessagesList";
+
```
Comment (Misha999777) [inline src/pages/MessagesPage.js:2]: One empty line is enough here

File: src/components/messages/Issue.js [hunk 19]
```diff
@@ -0,0 +1,25 @@
...
+            <div className="flex flex-col justify-center align-center">
+            <img src={error} alt="error" style={{width: '200px', height: '200px' }}/>
+            </div>
+        <div className="p-4 flex flex-col justify-center mx-auto">
+          <div className="flex items-center my-4">
+            <h2 className="text-4xl font-bold mb-2 text-left">Упс!</h2>
+          </div>
+          <p className="text-gray-700 text-xl text-left my-4">На жаль, Ваше оголошення від 30.04.2023 не пройшло перевірку.</p>
+          <p className="text-gray-700 text-xl text-left my-4">Причина: “Недостовірні дані оплати. Спробуйте знову та укажіть коректні дані.”</p>
+          <p className="text-gray-700 text-xl text-left my-4">Модератор: 012345</p>
```
Comment (Misha999777) [inline src/components/messages/Issue.js:18]: Lines are too long

File: src/components/messages/Verified.js [hunk 20]
```diff
@@ -0,0 +1,28 @@
...
+            <div className="flex flex-col justify-center align-center">
+            <img src={verified} alt="verified" style={{ width: '250px', height: '200px' }}/>
+            </div>
+        <div className="p-4 flex flex-col justify-center mx-auto">
+          <div className="flex items-center my-4">
+            <h2 className="text-4xl font-bold mb-2 text-left">Перевірено!</h2>
+          </div>
+          <p className="text-gray-700 text-xl text-left my-4">Ваше оголошення від 01.05.2023 пройшло перевірку.</p>
+          <p className="text-gray-700 text-xl text-left my-4">Подивитись: https://www.figma.com/file/429z7x135iGDmmGkkXGzkN/Project?type=design&node-id=531-1993&t=tTY2yWOfyaeX0WJz-4</p>
+          <p className="text-gray-700 text-xl text-left my-4">Модератор: 039355</p>
```
Comment (Misha999777) [inline src/components/messages/Verified.js:21]: Lines are too long

File: src/components/messages/Views.js [hunk 21]
```diff
@@ -0,0 +1,22 @@
...
+    <main className="container max-w-xl mx-auto my-15 px-4">
+      <div className="w-full bg-gradient-to-br from-sky-100 to-sky-300 rounded-3xl flex p-6">
+            <div className="flex flex-col justify-center align-center">
+            <img src={inf} alt="inf" style={{width: '230px', height: '200px' }}/>
+            </div>
+        <div className="p-4 flex flex-col justify-center mx-auto">
+          <div className="flex items-center my-4">
+            <h2 className="text-4xl font-bold mb-2 text-left">Перегляди</h2>
+          </div>
+          <p className="text-gray-700 text-xl text-left my-4">Ваше оголошення від 23.04.2023 переглянули 1 000 людей! Сподіваємось, що незабаром збір буде закрито.</p>
```
Comment (Misha999777) [inline src/components/messages/Views.js:15]: Line is too long

**Review by Misha999777 — APPROVED** [review]
LGTM, except for formatting

**Review by ArCheeq — APPROVED** [review]
Thanks for the corrections

**Pull request merged**